mod/metrics/lib/card/all_answer_query.rb

```diff
@@ -70,6 +70,10 @@ def limit
       @paging[:limit]
     end
 
+    def new_name_year
+      @new_name_year ||= year.to_s == "latest" ? Time.now.year : year
+    end
```

Comment:
without this there are lots of "+latest" cards.  (not sure that's a huge problem; the real issue was the #find_by_answer_id thing mentioned above.  but it seemed kinda wrong)